docs/source/features/packaging_output_models.md

```diff
@@ -46,6 +46,30 @@ The inference config file is a json file including `execution_provider` and `ses
 #### SampleCode
 Olive will only provide sample codes for ONNX model. Sample code supports 3 different programming languages: `C++`, `C#` and `Python`. And a code snippet introducing how to use Olive output artifacts to inference candidate model with recommended inference configurations.
 
+### AzureMLModels
+AzureMLModels packaging will register the output models to your Azure Machine Learning workspace. The asset name will be set as `<packaging_config_name>_<accelerator_spec>_<model_rank>`. For instance, if the output model is ONNX model and the packaging config is:
```

Comment:
Could you help add extra notes to tell user the rank is start from 1 not 0.

Reply:
Lol starting from 0 is a classic software engineer way of thinking xd. We do indicate the rank starts from 1 [here](https://microsoft.github.io/Olive/features/packaging_output_models.html#candidatemodels), but I can update this also.

Reply:
Thanks, at the first glance, I will think the rank number is something like id in distributed training where the rank starts from 0.